libraries/AP_DDS/AP_DDS_Client.h

```diff
@@ -51,8 +52,6 @@ class AP_DDS_Client
     // input and output stream
     uint8_t *input_reliable_stream;
     uint8_t *output_reliable_stream;
-    uxrStreamId reliable_in;
```

Comment:
Why were these moved away from the pointers to a different part of the header?

Reply:
done resolved